src/TestReactors.jl

```diff
@@ -10,7 +10,7 @@ spcs = phaseDict["phase"]["Species"]; #mechanism dictionaries index:  phaseDict[
 rxns = phaseDict["phase"]["Reactions"];
 solv = phaseDict["Solvents"][1];
 liq = IdealDiluteSolution(spcs,rxns,solv;name="phase",diffusionlimited=true) #Define the phase (how species thermodynamic and kinetic properties calculated)
-initialconds = Dict(["T"=>450.0,"P"=>1e5,"V"=>1.0e-6*1e6,"octane"=>6.154e-3*1e6,"oxygen"=>4.953e-6*1e6]) #Set simulation Initial Temp and Pressure
```

Comment:
Can you check if this needs fixed in any of the ipython notebook examples for liquids (ConstantTVDomain and ParameterizedTConstantVDomain)?

Reply:
I removed the initialconds P from ConstantTVDomain in IJulia example. The ParametrizedTConstantVDomain IJulia example doesn't have P in its initialconds.